app/models/shipit/user.rb

```diff
@@ -123,5 +129,20 @@ def identify_renamed_user!
     rescue Octokit::NotFound
       false
     end
+
+    def org_email?(email_address)
+      email_address.present? && email_address.split("@").last&.include?(org_domain)
+    end
+
+    def filter_org_emails(emails)
+      emails
+        .select { |email| org_email?(email.email) }
```

Comment:
`email.email` is a smell.

Reply:
Yeah, this was on the list of things I need to revisit. Even ignoring that, 'emails' and 'email' are both kinda wrong terms for describing the records (and their collection) returned from the Github API. I'll adjust them.

Reply:
I've adjusted it now.